kudu/Dockerfile

```diff
@@ -76,6 +76,15 @@ ENV PATH=$PATH:/opt/nodejs/9/bin
 
 EXPOSE 8181
 
+# Install ClamAV
+RUN apt-get install -y clamav
```

Comment:
Move this part to the `# Install dependencies` part each run adds another layer. We want to minimize the number of Docker Image layers.

Reply:
Done